packages/web-react/src/components/TextArea/__tests__/TextArea.test.tsx

```diff
@@ -76,4 +76,11 @@ describe('TextArea', () => {
       expect(element.style.height).toBe('');
     });
   });
+
+  it('should have correct id', () => {
```

Comment:
As you are repeating the `id` test in many places how about refactoring this to some kind of propsTest? See the `validationTextPropsTest` for example and you can create the `idPropsTest` and use it everywhere it is necessary to keep things DRY.

Reply:
requiredPropsTest has been added